bundles/org.openhab.binding.metofficedatahub/README.md

```diff
@@ -0,0 +1,472 @@
+Due to the current frequency of updates of the data retrieved, the hourly forecast data updates once per hour.
+The daily every 3 hours by default. 
+These are configurable but likely should only need to be lowered to reduce data if needed given the point below.
+
+**IMPORTANT:** The Met Office Data Hub service is free of charge for low volume users. 
+Higher data usages are charged, please see their website for current information.
+Please bear this in mind before adjust polling rates, or adding more than 1 location (site) for forecast data, as you may need a different plan depending on the data throughput over a month, or API hit rate.
+
+A possible use case could be to pull forecast data, for the next day to determine if storage heaters or underfloor heating should be pre-heated overnight.
+
+### Prerequisite
+
+In order to use this binding, you will need a Met Office Data Hub account. 
+Once created you will need to create a plan for access to the "Site Specific" subscriptions. 
+This will give you the client id and secret required for the bridge.
+
+## Supported Things
+
+This binding consists of a bridge for connecting to the Met Office Data Hub service with your account. 
+You can then add things to get the forecast's for a specific location (site), using this bridge.
+
+This binding supports the follow thing types:
+
+| Thing             | Thing Type | Thing Type UID  | Discovery | Description                                                                                 |
+|-------------------|------------|-----------------|-----------|---------------------------------------------------------------------------------------------|
+| Bridge            | Bridge     | bridge          | Manual    | A single connection to the Met Office DataHub API with daily poll limiting for the Site API |
+| Site Specific API | Thing      | siteSpecificApi | Manual    | Provides the hourly and daily forecast data for a give location (site)                      |
+
+## Thing Configuration
+
+### Bridge configuration parameters
+
+The bridge counts the total number of requests from 00:00 -> 23:59 under its properties during the runtime of the system.
+(This reset's if OH restarts, or the binding resets).
+
+| Name                       | Type   | Description                                                                                                                     | Default Values |
+|----------------------------|--------|---------------------------------------------------------------------------------------------------------------------------------|----------------|
+| siteSpecificRateDailyLimit | Number | For the runtime of the system, this is the limit of how many polls for updates are allowed for updates for the SiteSpecific API | 250            |
+| siteSpecificApiKey         | String | The API Key for the Site Specific subscription in your MET Office Data Hub account.                                             |                |
```

Comment:
I got rid of the "Specific" part of the names, but if the binding was extended to support other services offered by the same the met office, it may be unclear which one it relates to. So I have left site as the initial context to the setting as each one may be different should this happen. Hopefully that makes sense.

Reply:
It is the openHAB model and the names should be decoupled from the API. Obviously the user must know what key to put in what parameter. The description explains this very well, if you ask me the shorter name is better. Anyway, I leave it to you to decide wether you adapt this or not.

Reply:
Yeah if its okay lets keep the site prefix then, as if the other ones are added, then they may conflict without that extra pre-context, to separate them.

Reply:
Just to note: they will not conflict by convention as they will be added to a different Thing. Parameters only need to be unique for that Thing, not binding wide.